src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx

```diff
@@ -44,6 +44,7 @@ const GordonScheduleCalendar = ({ schedule, onSelectEvent }: Props) => {
         dayHeaderFormat: () => schedule.session.SessionDescription,
       }}
       onSelectEvent={onSelectEvent}
+      onKeyPressEvent={onSelectEvent}
```

Comment:
Looking at the docs for `react-big-calendar`, it appears that `onKeyPress` passes two parameters: firstly, the "calendar event" that was selected (i.e. the course in our case), and secondly the "keypress event", i.e. details about the keypress that occurred. (Event is such an overloaded word in programming...)

We can use the info on the keypress event to decide whether the key press is equivalent to selecting an event or not. For example:
```suggestion
      onKeyPressEvent={(selectedEvent, keyPressEvent) => {
          if (event.Key === 'Enter') {
              onSelectEvent(selectedEvent);
          }
      }}
```

Reply:
Tried looking at the ```react-big-calender``` documentation as well, didn't realize the "SyntheticEvent" mentioned as the second parameter was actually a keypress event. Thanks Evan!